### DISCUSSION TITLE
Small button size

### DISCUSSION THREAD

**Discussion by saskliutas** [iTwin/iTwinUI#2280] [Q&A] [answered]
Hi, I noticed that small button and small input have different height. Medium and large variants have same height. Is there a reason why height differ for small input and button?

This difference means that some additional alignment might be necessary when showing small input and button alongside. You can see height difference in this [sandbox](https://stackblitz.com/edit/github-ozu6pv?file=src%2FApp.tsx).

**Comment by mayank99** [comment, marked as answer]
Good catch, this is not intentional. They should both be the same height. We'll fix it in the next patch.

**Reply by mayank99** [reply]
This has been fixed in [3.15.3](https://github.com/iTwin/iTwinUI/releases/tag/%40itwin/itwinui-react%403.15.3) 🚀
